pkg/volume/util/nested_volumes.go

```diff
@@ -80,7 +80,7 @@ func getNestedMountpoints(name, baseDir string, pod v1.Pod) ([]string, error) {
 				}
 				// since this mount point is nested, remember it so that we can check that following ones aren't nested beneath this one
 				prevNestedMPs = append(prevNestedMPs, mp+string(os.PathSeparator))
-				retval = append(retval, mp[len(myMPSlash):])
+				retval = append(retval, mp[len(myMountPointWithSlash):])
```

Comment:
Not sure how these are related to sidecar image update?

Reply:
it is not. the first commit is just on GO readablility and not related to sidecar udpate. Considering it is cosmetic change (https://github.com/kubernetes/kubernetes/pull/134304/commits/c4ec18b58a6398084383035ad920ad3d7a047c200) , kept it as a seperate commit. @xing-yang